### ISSUE TITLE
Check this plugin for security issues

### ISSUE THREAD

**Issue by mervb** [easypropertylistings/epl-xml-csv-listing-import#64]

**Comment by joeguilmette** [comment]
If you find anything please shoot us an email if there's anything we can do to help.

**Comment by mervb** [comment]
Hi Joe,

Working on EPL core first as that powers this extension. Running it through phpcs --standard=wordpress 

BIG job, great tool wish we knew about this thing sooner to be honest. Once the core is done (we've been working on the importer 2.0 version on hold now till core is done)

**Comment by mervb** [comment]
complete
